Library/ViewModels/ProjectPageViewModel.swift

```diff
@@ -341,6 +341,66 @@ public final class ProjectPageViewModel: ProjectPageViewModelType, ProjectPageVi
         )
       }
 
+    // FB CAPI
+    let graphUser = self.viewDidAppearAnimatedProperty.signal.ignoreValues()
+      .switchMap { _ in
+        AppEnvironment.current.apiService
+          .fetchGraphUser(withStoredCards: false)
+          .ksr_delay(AppEnvironment.current.apiDelayInterval, on: AppEnvironment.current.scheduler)
+          .materialize()
+      }
```

Comment:
@msadoon  The reason this property is optional is because a non logged in user might view a project. However, if a logged in user does, we need to send the email. I'm not sure how I'd get the current user's email here without making the network request

Reply:
@scottkicks okay so as discussed yesterday we'll make this a separate ticket and an email we receive from a small one field query in `AppDelegate` should be accessible app-wide. `AppEnvironment.currency.loggedInUserEmail` For the purposes of this PR though let's remove `graphUser` signal and pass in `nil` as the `userEmail`